ui/perfherder/health/ItemBadge.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function ItemBadge(props) {
+  const { itemText, outlined, title } = props;
+
+  return (
+    <span
+      key={`item_${itemText}`}
+      className={`item-badge mr-1 ${outlined ? 'item-outlined' : ''}`}
+      title={title}
+    >
+      {itemText}
+    </span>
+  );
```

Comment:
Will this component will be going away based on your recent screenshot? (FYI, for future use of badges there's a react strap [component](https://reactstrap.github.io/components/badge/) for that).

Reply:
The first iteration used simple reactstrap badges (when it had colored background), but after that, I created the ItemBadge component to have the same feel of the simple badge, but with the outline(it does not come with outline in reactstrap). Now that I deleted the outline, I forgot to simplify the component (maybe remove it altogether). Thanks for pointing it out :+1: